### PR TITLE
Removed absolute styling from arrows

### PR DESCRIPTION
**Pull request by Kraig5**
Updated styling so that the navigation arrows are positioned relative to the videos (so they work on the service page and the proxy page.)

### CODE REVIEW

File: client/src/styles.css [hunk 1]
```diff
@@ -43,37 +45,33 @@ body {
...
-  overflow: hidden;
-  left:77%;
-  top: 27%;
+  top: 55px;
+  height: 60px;
 }
 
 .left.invisible {
-  visibility: hidden;
+  visibility: visible;
```
Comment (priscilagonzalez) [inline client/src/styles.css]: What happens to the arrow when the scrollbar reaches the end? Is visibility changed? They're supposed to be invisible.

File: client/src/styles.css [hunk 2]
```diff
@@ -3,15 +3,17 @@ body {
   font-family: Roobert,Helvetica Neue,Helvetica,Arial,sans-serif!important;
 }
 
+#app {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (priscilagonzalez) [inline client/src/styles.css]: Yes! This makes sense!

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -118,22 +118,22 @@ class App extends React.Component {
...
+        />
+        <div id="container" style={{ display: 'flex' }}>
           <div className={left} onClick={this.handleOnLeftClick}>
             <svg height="50" width="50">
               <circle id="circle1" cx="25" cy="25" fill="#18181b" r="25" />
               <text fill="#9147ff" fontFamily="Arial" fontSize="30px" x="35%" y="68%">{rightArrow}</text>
             </svg>
           </div>
-          {state.videos.map((video) => <Video video={video} />)}
+          <div id="scrollMe" style={{ display: 'flex', overflow: 'hidden' }}>
```
Comment (priscilagonzalez) [inline client/src/components/App.jsx]: This is already on style.css.

File: client/src/styles.css (hunk 1 above)
Comment (Kraig5) [reply client/src/styles.css]: Good catch. That was for my testing. I'll push a fix.

File: client/src/components/App.jsx (hunk 3 above)
Comment (Kraig5) [reply client/src/components/App.jsx]: Not sure what you mean? One of the problems I ran into was the arrows responding to the scrolling. In other words, when I would scroll right, the arrow would move too.  So I had to isolate the arrows from the videos, so I made the new "scrollMe" tag.

Comment (priscilagonzalez) [reply client/src/components/App.jsx]: Sorry I selected the wrong line. `I meant line 127 <div id="container" style={{ display: 'flex' }}>`

**Pull request merged**